.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is.-->
+
+**To Reproduce**
+<!-- Steps to reproduce the behaviour:
+
+1. happens with input (include input files)
+2. which version of DFTB+ is used (downloaded, conda-forge or build from source)
```

Comment:
or, if building from source, the relevant git commit and the compiler and libraries used)